data/os/CentOS.yaml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
we can yank this file because RedHat.yaml matches for Centos/Rock/Alma systems as well, because of the identical OS family

Reply:
The URL is different: `https://repos.influxdata.com/rhel` vs. `https://repos.influxdata.com/centos` thats why I go with the name here.

Reply:
Right, they do actually have different base URLs for RedHat and CentOS.  I don't know if it would cause any actual failures if we removed this file, but I think if they offer them we might as well use them.